app/controllers/conference_controller.rb

```diff
@@ -13,8 +13,10 @@ def show; end
 
   def schedule
     @rooms = @conference.venue.rooms if @conference.venue
-    @events = @conference.program.events
-    @events_xml = @events.scheduled.order(start_time: :asc).group_by{ |event| event.start_time.to_date }
+    program = @conference.program
+    @events = program.events
+    @events_xml = program.schedules.find(program.selected_schedule).event_schedules.where('start_time IS NOT NULL AND room_id IS NOT NULL').order(start_time: :asc)
```

Comment:
schedule controller is in the admin folder. Everything related with the schedule is now in conference controller. Do you want to change that?